minimal.Dockerfile

```
@@ -0,0 +1,61 @@
+# Steps to build and push minimal SMARTS docker image
+# ```bash
+# $ export VERSION=v0.4.13
+# $ cd /path/to/SMARTS
+# $ docker build -t huaweinoah/smarts:$VERSION-minimal -f minimal.Dockerfile .
+# $ docker push huaweinoah/smarts:$VERSION-minimal
+# ```
+
+FROM ubuntu:bionic
+
+# Install libraries
+RUN apt-get update --fix-missing && \
```

OK, the other option:

```suggestion
RUN export DEBIAN_FRONTEND=noninteractive && \
    apt-get update --fix-missing && \
```